uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/dnd/GridWidgetDnDMouseMoveHandler.java

```diff
@@ -181,10 +184,9 @@ private void setCursor(final Style.Cursor cursor) {
     }
 
     protected void findResizableColumn(final GridWidget view,
+                                       final BaseGridRendererHelper.RenderingInformation renderingInformation,
                                        final double cx) {
         //Gather information on columns
-        final BaseGridRendererHelper rendererHelper = view.getRendererHelper();
-        final BaseGridRendererHelper.RenderingInformation renderingInformation = rendererHelper.getRenderingInformation();
```

Comment:
Is this `renderingInformation` different from the one now obtained as parameter? If it is different I think we should remove `getRenderingInformation()` from `BaseGridRendererHelper` because it is not reliable (and maybe `getRendererHelper()` from `view` too?)

Reply:
No it is exactly the same (given the same state of the grid, position, number of rows, columns etc). All calls to `BaseGridRendererHelper.getRenderingInformation(..)` will return identical results (although the result is is not cached and is re-created on each invocation). So passing an existing instance around removes the need to re-create it over and over again.